pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -76,6 +78,7 @@ type Config struct {
 		TokenBody   string `gcfg:"token-body"`
 		ProjectID   string `gcfg:"project-id"`
 		NetworkName string `gcfg:"network-name"`
+		Multizone   string `gcfg:"multizone"`
```

Comment:
My intention here was to allow additional behaviours in future.  e.g. directly specifying the managed zones, rather than managing all the zones in a region.  That said, if we can auto-configure based on node registration, this requirement goes away.  And maybe it should just be a bool anyway for now, and we can add a list of zones if/when we need it.